### PR TITLE
Change examples to use jar from maven central and test

### PR DESCRIPTION
**Pull request by ravjotbrar**
### Summary

Added library dependency for assembly JAR for connector from maven central.

### Description

When users were running the examples, they had to manually add a lib folder along with the JAR file. This step is now removed by adding the library dependency from maven central in the build.sbt file for each example. This change is also reflected in the READ.md file in the examples folder. 

### Related Issue

http://jira.verticacorp.com:8080/jira/browse/VER-77164

### Additional Reviewers

@alexr-bq 
@jonathanl-bq 
@alexey-temnikov

### CODE REVIEW

File: examples/demo/build.sbt [hunk 1]
```diff
@@ -20,3 +20,4 @@ resolvers += "Artima Maven Repository" at "https://repo.artima.com/releases"
 resolvers += "jitpack" at "https://jitpack.io"
 
 libraryDependencies += "com.typesafe" % "config" % "1.4.1"
+libraryDependencies += "com.vertica.spark" % "vertica-spark" % "2.0.0-rc0"
```
Comment (alexr-bq) [inline examples/demo/build.sbt:23]: May as well make this consistent with the others and use the Seq

**Review by alexr-bq — APPROVED** [review]
Looks good! Sign the CLA

**Comment by alexr-bq** [comment]
Just to confirm: you tested this with nothing in lib/ dir?

File: examples/demo/build.sbt (hunk 1 above)
Comment (ravjotbrar) [reply examples/demo/build.sbt]: Done

**Comment by ravjotbrar** [comment]
> Just to confirm: you tested this with nothing in lib/ dir?

That's right. No lib directory in any of the examples.

File: examples/README.md [hunk 2]
```diff
@@ -35,4 +24,4 @@ docker exec docker_hdfs_1 /opt/hadoop/sbin/start-dfs.sh
 
 Run `docker exec -it docker_client_1 /bin/bash` to enter the sandbox client environment.
 
-Now just run `sbt run` from the `/spark-connector/examples/demo` directory.
+Now just run `sbt "run [CASE]"` from the `/spark-connector/examples/demo` directory.
```
Comment (jonathanl-bq) [inline examples/README.md:27]: We should be more general and state how to run any of the examples, not just the cases from the demo example.

**Review by jonathanl-bq — CHANGES_REQUESTED** [review]

File: examples/README.md (hunk 2 above)
Comment (ravjotbrar) [reply examples/README.md:27]: What I'm thinking is to use the basic-write example as a case in the demo example. We already have a basicRead case, so the basic-read example seems redundant. This way, all specific examples/cases are organized in one directory. What do you think?

Comment (jonathanl-bq) [reply examples/README.md:27]: I'm okay with combining some of the other examples into the demo project, though I'm not sure what @alexr-bq thinks. I'm more concerned that there's now only instructions on how to run examples from the demo project. The new README doesn't explain how to run the Kerberos example, or any other future examples we might add (e.g. S3, TLS, etc.).

Comment (ravjotbrar) [reply examples/README.md:27]: I'll change that in the ticket that involves updating the documentation of the README.

Comment (jonathanl-bq) [reply examples/README.md:27]: Okay, fair enough.

**Review by jonathanl-bq — APPROVED** [review]

**Pull request merged**